api/src/wfl/api/workloads.clj

```diff
@@ -36,8 +36,9 @@
   (try
     (load-workload-impl tx workload)
     (catch Throwable cause
-      (throw (ex-info "Error loading workload"
-                      {:workload workload} cause)))))
+      (throw (ex-info (str "Error loading workload - " (.getMessage cause))
+                      (util/deep-merge {:workload workload} (ex-data cause))
+                      cause)))))
```

Comment:
Why merge ex-data with the workload?

Reply:
If loading the workflow fails, the exception might have a :type or some other debugging data in the `ex-data`. The `ex-data` is what ends up getting returned to the user (or logged in the case of a 500-series) so I figure including the workload itself as context is helpful. It would be under its own key so hopefully it wouldn't be too unreadable. I'd be happy to take it out if you think it isn't necessary

Reply:
Ah. That's what threw me. Maybe just `(assoc (ex-data cause) :workload workload)` then?

Reply:
Fair enough, good idea